### PR TITLE
ebtables: Fix #3742

### LINKED ISSUE #3742: [ebtables] segmentation fault
ebtables-2.0.10.4_4 seq faults on void linux (glibc) x86_64

downgrading to ebtables-2.0.10.4_3.x86_64.xbps fixes the issue.

### PR DESCRIPTION
**Pull request by ebfe**

### CODE REVIEW

File: srcpkgs/ebtables/template [hunk 1]
```diff
@@ -24,6 +24,16 @@ pre_build() {
 	rm include/linux/if_ether.h
 }
 
+do_build() {
+	LDFLAGS=${LDFLAGS/--as-needed/--no-as-needed}
```
Comment (Gottox) [inline srcpkgs/ebtables/template]: use `pre_build()` and remove everything below.

Comment (ebfe) [inline srcpkgs/ebtables/template]: > use `pre_build()` and remove everything below.
done

File: srcpkgs/ebtables/template [hunk 2]
```diff
@@ -22,6 +22,7 @@ make_install_args="
 
 pre_build() {
 	rm include/linux/if_ether.h
+	LDFLAGS=${LDFLAGS/--as-needed/--no-as-needed}
```
Comment (xtraeme) [inline srcpkgs/ebtables/template]: Use `LDFLAGS="-Wl,--no-as-needed"` in the main block instead.

**Comment by xtraeme** [comment]
:+1:

**Comment by ebfe** [comment]
0b97e9adc489a9658e589e0d42b5538f8f74a636

**Pull request closed without merging**